File: client-node/src/main/scala/pl/jaca/ircsy/clientnode/connection/ConnectionDesc.scala
package pl.jaca.ircsy.clientnode.connection

/**
  * @author Jaca777
  *         Created 2016-05-02 at 16
  */
case class ConnectionDesc(serverDesc: ServerDesc, username: String) extends Serializable{
  override def toString =  serverDesc + "@" + username
}

File: chat/src/main/java/pl/jaca/ircsy/chat/ConnectionDesc.java
package pl.jaca.ircsy.chat;

/**
 * @author Jaca777
 *         Created 2016-05-28 at 16
 */
public class ConnectionDesc {
    private ServerDesc server;
    private String nickname;

    protected ConnectionDesc(){

    }

    public ConnectionDesc(ServerDesc server, String nickname) {
        this.server = server;
        this.nickname = nickname;
    }

    public ServerDesc getServer() {
        return server;
    }

    public String getNickname() {
        return nickname;
    }

    @Override
    public String toString() {
        return server + "@" + nickname;
    }
}
